tools/accuracy_checker/accuracy_checker/annotation_converters/nyu_depth.py

```diff
@@ -28,33 +28,44 @@ def parameters(cls):
             'data_dir': PathField(
                 is_directory=True, optional=True,
                 description='path to directory with data in original hdf5 format stored'
+            ),
+            'allow_convert_data': BoolField(
+                optional=True, default=True, description="Allows to convert data from hdf5 format"
             )
         })
         return parameters
 
     def configure(self):
         self.data_dir = self.get_value_from_config('data_dir')
-        if self.data_dir is None and not contains_all(self.config, ['images_dir', 'depth_map_dir']):
-            raise ConfigError('data_dir or both images_dir and depth_map_dir should be provided')
+        self.allow_convert_data = self.get_value_from_config('allow_convert_data')
         self.images_dir = self.get_value_from_config('images_dir')
         self.depths_dir = self.get_value_from_config('depth_map_dir')
-        if self.data_dir and isinstance(h5py, UnsupportedPackage):
-            h5py.raise_error(self.__provider__)
-        if self.images_dir is None:
-            self.images_dir = self.data_dir.parent / 'converted/images'
-        if self.depths_dir is None:
-            self.depths_dir = self.data_dir.parent / 'converted/depth'
-        if self.data_dir is None:
-            self.images_dir = get_path(self.images_dir, is_directory=True)
-            self.depths_dir = get_path(self.depths_dir, is_directory=True)
-        else:
+
+        if self.allow_convert_data:
+            if self.data_dir is None:
+                raise ConfigError('please provide data_dir to convert data from hdf5 format')
+            if isinstance(h5py, UnsupportedPackage):
+                h5py.raise_error(self.__provider__)
```

Comment:
logically, even user specify data dir, but does not have installed dependencies, he will not be able to convert, so h5py should be checked before data_dir
```suggestion
           if isinstance(h5py, UnsupportedPackage):
                h5py.raise_error(self.__provider__)
            if self.data_dir is None:
                raise ConfigError('please provide data_dir to convert data from hdf5 format')
```

Reply:
BTW, now specifying all 3 paths looks reductant, possibly this moment also should be refactored

Reply:
Done